<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1040</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpTiMSoC Host GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitterBottomSection">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="taskTabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="ConnectionView" name="tabConnection">
        <attribute name="title">
         <string>Connection</string>
        </attribute>
       </widget>
       <widget class="SystemView" name="tabSystem">
        <attribute name="title">
         <string>System Overview</string>
        </attribute>
       </widget>
       <widget class="SoftwareExecutionView" name="tabSoftwareExecution">
        <attribute name="title">
         <string>Software Execution</string>
        </attribute>
       </widget>
      </widget>
      <widget class="LogWidget" name="logViewer" native="true"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnToggleLogViewer">
        <property name="text">
         <string>Logs</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTarget_System">
    <property name="title">
     <string>&amp;Target System</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionStartCpus"/>
    <addaction name="actionStallCpus"/>
    <addaction name="actionResetSystem"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTarget_System"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>24</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartCpus"/>
   <addaction name="actionStallCpus"/>
   <addaction name="actionResetSystem"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="optimsocgui.qrc">
     <normaloff>:/resources/network-connect.png</normaloff>:/resources/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="optimsocgui.qrc">
     <normaloff>:/resources/network-disconnect.png</normaloff>:/resources/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionStartCpus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="optimsocgui.qrc">
     <normaloff>:/resources/media-playback-start.png</normaloff>:/resources/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start CPUs</string>
   </property>
   <property name="toolTip">
    <string>(Re-)Start CPUs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionResetSystem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset System</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="optimsocgui.qrc">
     <normaloff>:/resources/configure.png</normaloff>:/resources/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionExecution_Chart">
   <property name="text">
    <string>Execution Chart</string>
   </property>
  </action>
  <action name="actionInitialize">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Initialize</string>
   </property>
  </action>
  <action name="actionStallCpus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="optimsocgui.qrc">
     <normaloff>:/resources/media-playback-pause.png</normaloff>:/resources/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stall CPUs</string>
   </property>
   <property name="toolTip">
    <string>Stall CPUs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionView</class>
   <extends>QWidget</extends>
   <header>connectionview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>logwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SystemView</class>
   <extends>QWidget</extends>
   <header>systemview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SoftwareExecutionView</class>
   <extends>QWidget</extends>
   <header>softwareexecutionview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="optimsocgui.qrc"/>
 </resources>
 <connections/>
</ui>
